clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -49,6 +49,7 @@
 // DPCT_LABEL_END
 #include "functional.h"
 #include "iterators.h"
+#include "vector.h"
```

Comment:
How are vectors being used in this commit?  I couldn't find a use

Reply:
I've added the missing dependencies now, sorry about that.